packages/components/src/focal-point-picker/README.md

```diff
@@ -0,0 +1,68 @@
+# Focal Point Picker
+
+Focal Point Picker is a component which creates a UI for identifying the most important visual point of an image. This component addresses a specific problem: with large background images it is common to see undesirable crops, especially when viewing on smaller viewports such as mobile phones. This component allows the selection of the point with the most important visual information and returns it as a pair of numbers between 0 and 1. This value can be easily converted into the CSS `background-position` attribute, and will ensure that the focal point is never cropped out, regardless of viewport.
+
+Example focal point picker value: `{ x: 0.5, y: 0.1 }`
+Corresponding CSS: `background-position: 50% 10%;`
+
+## Usage
+
+```jsx
+import { FocalPointPicker } from '@wordpress/components';
+
+const MyFocalPointPicker = withState( {
+	focalPoint: {
+		x: 0.5,
+		y: 0.5
+	},
+} )( ( { focalPoint, setState } ) => { 
+	const url = '/path/to/image';
+	const dimensions = {
+		width: 400,
+		height: 100
+	};
+	return ( 
+		<FocalPointPicker 
+			url={ url }
+			dimensions={ dimensions }
+			value={ focalPoint }
+			onChange={ ( focalPoint ) => setState( { focalPoint } ) } 
+		/>
+	) 
+} );
+
+/* Example function to render the CSS styles based on Focal Point Picker value */
+const renderImageContainerWithFocalPoint = ( url, focalPoint ) => {
```

Comment:
This is meant to illustrate how the percentage values from the component can be translated into a "real-world" CSS `background-position` definition. The relevant line in Cover block is:

https://github.com/WordPress/gutenberg/blob/ded1017374c1bb815ba02ea7c04637be5dbf981b/packages/block-library/src/cover/index.js#L241

It would probably make sense for me to use the actual example instead of the hypothetical, but I'd prefer to hold off on updating the README until the PR is ready to merge, in case there are changes still to come. Would this be a reasonable approach?